pandas/core/groupby/groupby.py

```diff
@@ -1256,7 +1257,9 @@ def groupby_function(name, alias, npfunc,
                              numeric_only=True, _convert=False,
                              min_count=-1):
 
-            _local_template = "Compute %(f)s of group values"
+            _local_template = """
```

Comment:
Similar comment here

Reply:
Good call, this one is problematic and is leading to `reference/groupby.html` having descriptions like "Compute first of group values See Also ——– pandas.Series.groupby pandas.DataFrame.groupby pandas.Panel.groupby" rather than simply "Compute first of group values". I will take a look into this.

Reply:
@WillAyd I changed the formatting to add an extra newline to the docstring to prevent the two appender values from being put on a single line, but this causes another build failure, that the docstring ends with two newlines. Any advice on the proper way to resolve this? What should the code look like?